feature/experimental/tunnel/otg_tests/tunnel_interface_based_ipv4_gre_encapsulation_test/README.md

```diff
@@ -0,0 +1,87 @@
+# TUN-1.3: Interface based IPv4 GRE Encapsulation
+
+## Summary
+
+Validate Interface based Ipv4 GRE Tunnel Config.
+
+## Procedure
+
+Validate the GRE configuration
+GRE Tunnel interfaces configuration options
+Tunnel source
+Tunnel source should be able to configure with unnumbered interface address
+Tunnel Destination
+Directly on Tunnel interface or Tunnel Group
+Configure such 32 tunnel interfaces
+Configure static route with IPv4 NH pointing to Tunnel interfaces
+Send 1000 IPv4 flows from tester on ATE-PORT1 connected to DUT-PORT1 where GRE Encap filter is applied with the traffic profile documented
+IPv4 flows are made of 5 tuples
+Source IP address
+Source port
+Destination IP address
+Destination port
+Payload protocol
```

Comment:
Please delete. This test is for interface based tunneling where we are targeting tunnel interface based on the static route. Filter shouldnt play any role.

We will need new tests where we have L4 and other parameters matched in a filter and forwarded to the tunnel interface for encap and no route lookup required for inner header. Route lookup will only happen for the destination address in the outer header.

Reply:
Addressed comment.